collector/container/Dockerfile

```diff
@@ -1,17 +1,45 @@
 ARG BASE_REGISTRY=registry.access.redhat.com
 ARG BASE_IMAGE=ubi8/ubi-minimal
 ARG BASE_TAG=8.6
+
+ARG collector_version=xxx
+ARG module_version=xxx
+
 FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG} AS extracted_bundle
 
-COPY bundle.tar.gz /
-COPY extract-bundle.sh /bundle/
+ARG collector_version
+ARG module_version
+ARG ROOT_DIR=.
+
+ENV ROOT_DIR=$ROOT_DIR
+ENV COLLECTOR_VERSION=$collector_version
+ENV MODULE_VERSION=$module_version
+
+COPY . /go/src/github.com/stackrox/collector
+WORKDIR /go/src/github.com/stackrox/collector
+
+RUN mkdir /bundle
+
+RUN cp "${ROOT_DIR}/bundle.tar.gz" /
+RUN cp "${ROOT_DIR}/extract-bundle.sh" /bundle/
+RUN cp "${ROOT_DIR}"/scripts/* /
+
 WORKDIR /bundle
 RUN ./extract-bundle.sh
 
+RUN echo "${collector_version}" > /COLLECTOR_VERSION
```

Comment:
I think this will always overwrite the `COLLECTOR_VERSION` file in `.openshift-ci/build/Dockerfile.create_bundle`